fbfmaproom/fbfmaproom.py

```diff
@@ -220,7 +220,7 @@ def open_data_array(
             val_max = cfg["range"][1]
         else:
             assert False, "configuration doesn't specify range"
-    da.attrs["colormap"] = cfg["colormap"]
+    da.attrs["colormap"] = CMAPS[cfg["colormap"]]
```

Comment:
This is fine, but if you decide that the PNE colorscale should be FBF-specific, you can do something like
```
import pingrid
CMAPS = dict(pingrid.CMAPS, {
    'pne': pingrid.ColorScale(
        ...
    )
}
```